rdkit/Chem/Descriptors.py

```diff
@@ -88,6 +88,36 @@ def _setupDescriptors(namespace):
   31.05...
 
 """
+def GetNumChiralCenters(mol):
+  """The number of chiral centers in a molecule
+  >>>> GetNumChiralCenters(Chem.MolFromSmiles('CC(N)O'))
+  1
+  >>>> GetNumChiralCenters(Chem.MolFromSmiles('CC(Br)C(N)O'))
+  2
+  
+  """
+  return len(Chem.FindMolChiralCenters(mol, includeUnassigned=True))
+  
+
+
+def GetNumSpecifiedChiralCenters(mol):
+  """Number of specified chiral centers in a molecule
+  >>>>  GetNumSpecifiedChiralCenters(Chem.MolFromSmiles('C[C@H](N)O'))
```

Comment:
Ah, ok. That might have been one of the few things I hadn't considered. Thanks for the help.